src/node_api.cc

```diff
@@ -570,8 +570,7 @@ class SetterCallbackWrapper
 
   /*virtual*/
   void SetReturnValue(napi_value value) override {
-    node::FatalError("napi_set_return_value",
-      "Cannot return a value from a setter callback.");
+    assert(false);  // Cannot return a value from a setter callback
```

Comment:
@jasongin I think this might be something that really surprises people when they set out to debug an issue in their addon and suddenly have to deal with more, unexpected, crashes that weren’t there in release mode…